lib/utils.js

```diff
@@ -18,32 +19,54 @@ async function getPIDsUsingPattern (pattern, opts = {}) {
   const args = [`-${ignoreCase ? 'i' : ''}f${multi ? '' : 'n'}`, pattern];
   try {
     const {stdout} = await exec('pgrep', args);
-    if (multi) {
-      const result = stdout.split('\n')
-        .filter((x) => parseInt(x, 10))
-        .map((x) => `${parseInt(x, 10)}`);
-      return _.isEmpty(result) ? null : result;
-    }
-    const pid = parseInt(stdout, 10);
-    return isNaN(pid) ? null : `${pid}`;
+    return stdout.split(/\s+/)
```

Comment:
This function is called in one place (in `killAppUsingPattern`, and always with `opts.multi`and `opts.ignoreCase` set to `true`. So why have those options at all? If ever needed, this could easily be added.

At the very least, the defaults should be those actually used, so that the calling code doesn't need to pass in the options.

Reply:
makes sense. will change